addons/web/static/src/search/search_model.js

```diff
@@ -1005,6 +1008,23 @@ export class SearchModel extends EventBus {
         this._notify();
     }
 
+    async spawnCustomFilterDialog() {
+        const fieldDefs = await this.fieldService.loadFields(this.resModel);
```

Comment:
I think fields are already known in search model

Reply:
this.searchViewFields

Reply:
A nice (side) ref would be to accept domain as optional in DomainSelector(Dialog) and compute the default there? Same thing for TreeEditor with tree?

Reply:
@Polymorphe57 Using this.searchViewFields failed a bunch of tests.
I put back this version to make sure the runbot is ok.
We can investigate why it fails tomorrow.